exercises/practice/book-store/lib/book_store.ex

```diff
@@ -0,0 +1,10 @@
+defmodule BookStore do
+  @type book :: integer
```

Comment:
It took me a while to realize why a book is an integer. Maybe something more explicit with a typedoc could help?
```suggestion
  @typedoc "A book is represented by its number in the 5-book series"
  @type book :: 1 | 2 | 3 | 4 | 5
```